railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -58,6 +58,8 @@ Rails.application.configure do
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
+  # Skip http-to-https redirect for the default health check endpoint.
+  # config.ssl_options = { redirect: { exclude: ->(request) { request.path == "/up" } } }
```

Comment:
Would `Rails.application.routes.url_helpers.rails_health_check_path` make sense here?

Reply:
I think it would force evaluating the routes earlier. So not great.